src/lib/components/wizard-navigation-bar.component.vertical.less

```diff
@@ -243,7 +243,7 @@ aw-wizard-navigation-bar.vertical {
           .state-circle-with-border-hover(@dot-border-width, @wz-color-default);
         }
 
-        div {
+        a {
           min-height: @big-dot-height + 2 * @dot-border-width;
```

Comment:
The height seems to be a little to large for the empty circle layout with and without symbol.
I believe that the `+ 2 * @dot-border-width` is not required anymore with the new div based step indicators.

Reply:
I forgot to add an example image:
![grafik](https://user-images.githubusercontent.com/18488086/46961255-aec08f00-d0a0-11e8-9dd1-7b7d2beda277.png)
On the image you can see that the labels are slightly lower positioned than the symbols in the step indicators.

Reply:
Screenshot from the updated version (388a9777):

![updated label alignment in vertical layout](https://user-images.githubusercontent.com/52021/47092153-0856c400-d22f-11e8-9109-1a59137de5f7.png)